File: RealEstateManagementApp/Models/MessageDto.cs
namespace RealEstateManagementApp.Models
{
    public class MessageDto
    {
        public string? msgText { get; set; }
        public string? chatId { get; set; }
        public int? tenantId { get; set; }
        public int? landlordId { get; set; }
        public DateTime? sentTime { get; set; }
        public int? sentByUserId { get; set; }
    }
}

File: RealEstateAngular/src/app/models/MessageDto.ts
import { DateSelectionModelChange } from "@angular/material/datepicker";

export class MessageDto {
    public msgText: string = '';
    public chatId?: string;
    public tenantId?: number;
    public landlordId?: number;
    public sentTime?: Date;
    public sentByUserId?: number;
  }